ipaserver/install/httpinstance.py

```diff
@@ -345,79 +277,32 @@ def configure_certmonger_renewal_guard(self):
             if certmonger_stopped:
                 certmonger.stop()
 
-    def create_password_conf(self):
-        """
-        This is the format of mod_nss pin files.
-        """
-        pwd_conf = paths.HTTPD_PASSWORD_CONF
-        ipautil.backup_file(pwd_conf)
-
-        passwd_fname = os.path.join(paths.HTTPD_ALIAS_DIR, 'pwdfile.txt')
-        with open(passwd_fname, 'r') as pwdfile:
-            password = pwdfile.read()
-
-        with open(pwd_conf, "w") as f:
-            f.write("internal:")
-            f.write(password)
-            f.write("\nNSS FIPS 140-2 Certificate DB:")
-            f.write(password)
-            # make sure other processes can access the file contents ASAP
-            f.flush()
-        pent = pwd.getpwnam(constants.HTTPD_USER)
-        os.chown(pwd_conf, pent.pw_uid, pent.pw_gid)
-        os.chmod(pwd_conf, 0o400)
-
-    def disable_system_trust(self):
-        name = 'Root Certs'
-        args = [paths.MODUTIL, '-dbdir', paths.HTTPD_ALIAS_DIR, '-force']
-
-        try:
-            result = ipautil.run(args + ['-list', name],
-                                 env={},
-                                 capture_output=True)
-        except ipautil.CalledProcessError as e:
-            if e.returncode == 29:  # ERROR: Module not found in database.
-                logger.debug(
-                    'Module %s not available, treating as disabled', name)
-                return False
-            raise
-
-        if 'Status: Enabled' in result.output:
-            ipautil.run(args + ['-disable', name], env={})
-            return True
-
-        return False
-
     def __setup_ssl(self):
         db = certs.CertDB(self.realm, nssdir=paths.HTTPD_ALIAS_DIR,
                           subject_base=self.subject_base, user="root",
                           group=constants.HTTPD_GROUP,
                           create=True)
-        self.disable_system_trust()
-        self.create_password_conf()
 
         if self.pkcs12_info:
-            if self.ca_is_configured:
```

Comment:
If you want, I can take care of the CA-less scenario here.

Reply:
That would be awesome and fits my plan to pull other devs in to complete the remaining tasks, thanks!